Classes/WPEditorView.m

```diff
@@ -1262,9 +1274,11 @@ - (void)updateLink:(NSString *)url
     
     url = [self normalizeURL:url];
     
-	NSString *trigger = [NSString stringWithFormat:@"ZSSEditor.updateLink(\"%@\",\"%@\");", url, title];
-	[self.webView stringByEvaluatingJavaScriptFromString:trigger];
-	
+    if (self.isInVisualMode) {
```

Comment:
Well, since this shouldn't be called at all outside of visual mode... I'd stick an `NSAssert(self.isInVisualMode, @"...");` instead of the `if` block (or in addition if you prefer, but then you can wrap all code lines inside the `if` block since nothing should really execute).